src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/AwsDatasetModification.kt

```diff
@@ -120,6 +120,7 @@ class AwsDatasetModification private constructor(
             aws = aws
         )
 
+        fun onlineTransformation(onlineTransformation: Consumer<Infrastructure<*>>) = apply { this.onlineTransformation = onlineTransformation }
```

Comment:
I like it
We should include the rationale in the commit message

Reply:
![image](https://user-images.githubusercontent.com/1194704/60431093-32a2b300-9bff-11e9-90f6-0d8ecdec7430.png)
Few things here:
* the summary is too long and is being wrapped, imho "Refactor `AwsDatasetModification` summarizes the commit well enough
* `enchance` -> `enhance`
* `extention` -> `extension`